packages/api/endpoints/granules.js

```diff
@@ -711,7 +715,8 @@ router.get('/:granuleName', get);
 router.get('/', list);
 router.post('/:granuleName/executions', associateExecution);
 router.post('/', create);
-router.put('/:granuleName', put);
+router.put('/:granuleName', patch);
```

Comment:
🤔 I see why we're doing this but 🤔. We don't want to break existing update calls (although users will need to know what PUT _actually_ does). 

I'm torn. It seems like we might want to have this break rather than have a PUT call actually PATCH.

Reply:
We're planning on doing that in CUMULUS-3072, this PR is designed to be merged as a compatible change making PATCH available for use in our code without requiring a user change/api version/etc.

Reply:
Yup, discussed on standup. Makes sense to me!